<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acedemic</class>
 <widget class="QMainWindow" name="acedemic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#d3ffc5;   /* Mint green */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1091</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget {
    border: 2px solid #007BFF;      /* Blue border */
    background-color: #f4f4f4;      /* Light gray background */
    padding: 10px;                   /* Padding around the widget */
    border-radius: 5px;              /* Rounded corners */
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>76</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sunday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thursday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Friday</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>430</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>add this for every week</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter this details one by one and submit one by one</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>118</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>subject</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>hours</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>day</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 28px;            /* Larger font size for emphasis */
    color: #4CAF50;             /* Green color for a fresh look */
    font-weight: bold;          /* Make the text bold */
    padding: 10px 20px;         /* Add padding around the text */
    background-color: #e8f5e9;  /* Light green background to match the color scheme */
    border-radius: 8px;         /* Rounded corners */
    text-align: center;         /* Center-align the text */
}
</string>
      </property>
      <property name="text">
       <string>Basic Acedemic Schedule</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 25px;
    color: #333333;   /* Dark gray color for the text */
    padding: 5px;
    font-weight: normal;  /* Regular weight */
    text-align: left;     /* Align the text to the left */
}
</string>
      </property>
      <property name="text">
       <string>Enter your lecture details:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>venue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Save1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>470</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Back1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>470</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>390</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GoToNextPage">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>270</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>go to next page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="Save2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>deadline:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>501</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 32px;            /* Larger font size for emphasis */
    color: #4CAF50;             /* Green color for a fresh look */
    font-weight: bold;          /* Make the text bold */
    padding: 10px 20px;         /* Add padding around the text */
    background-color: #e8f5e9;  /* Light green background to match the color scheme */
    border-radius: 8px;         /* Rounded corners */
    text-align: center;         /* Center-align the text */
}
</string>
      </property>
      <property name="text">
       <string>Advanced Acedemic Schedule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Back2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>440</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>392</width>
        <height>236</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>118</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GoToPreviousPage">
      <property name="geometry">
       <rect>
        <x>892</x>
        <y>250</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>go to previous page</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 25px;            /* Larger font size for emphasis */
    color: #4CAF50;             /* Green color for a fresh look */
    font-weight: bold;          /* Make the text bold */
    padding: 10px 20px;         /* Add padding around the text */
    background-color: #e8f5e9;  /* Light green background to match the color scheme */
    border-radius: 8px;         /* Rounded corners */
    text-align: center;         /* Center-align the text */
}
</string>
    </property>
    <property name="text">
     <string>Acedemic schedule</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
